Tests/SwiftPrettyPrintTests/Core/URLColoringTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  URLColoringWithDefaultColorThemeTests.swift
+//  SwiftPrettyPrintTests
+//
+//  Created by ogasawara on 2021/02/08.
+//
+
+@testable import SwiftPrettyPrint
+import XCTest
+import SwiftParamTest
+import Curry
+
+/// Test the coloring of URL.
+class URLColoringTests: XCTestCase {
+    let exampleURL = URL(string: "https://example.com")!
+
+    override func setUp() {}
+
+    override func tearDown() {}
+    
+    ///
+    /// URL Coloring with Custom Color Theme
+    ///
+    func testWithCustomColorTheme() {
+        let theme: ColorTheme = {
+            var t = ColorTheme.plain
+            t.url = { #"<a href="\#($0)">\#($0)</a>"# }
```

Comment:
I feel `a` tag with `href` attribute are redundant little for this test.
I'll consider it at organize test of `ColorTheme`.

Reply:
I understand.